tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp

```diff
@@ -46,6 +46,45 @@ void test_suite_for_frustum(const bool with_equiangular_map) {
   CAPTURE(upper_x_upper_base);
   const double upper_y_upper_base = upper_bound_upper_base_dis(gen);
   CAPTURE(upper_y_upper_base);
+  const double upper_z = 3.0;
+  CAPTURE(upper_z);
+  const double lower_z = -1.0;
+  CAPTURE(lower_z);
+
+  // For diagnostic purposes, compute other Frustum quantities:
+  // Frustum cross factor is small when the diagonals across each of the bases
+  // are aligned, and large when there is an angle between them.
```

Comment:
I like how you gave guidance with the other parameters whether large/small could be considered "bad". Is there such guidance available for `frustum_cross`?